index.html

```diff
@@ -204,26 +219,36 @@ <h1>About me</h1>
             </div>
         </section>
         <section class="comments">
-            <h2>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h2>
-            <form class="form_comment" action="https://formspree.io/f/xdovrwnp" method="post">
-                <input type="text" id="fname" name="fname" placeholder="Full Name" minlength="4" maxlength="30" class="input_place input_comment" required>
-                <hr class="InputSeparation">
-                <br>
-                <br>
-                <br>
-                <input id="faddress" name="faddress" class="input_place input_comment" placeholder="Email Address" type="email" minlength="10" maxlength="30" required>
-                <hr class="InputSeparation">
-                <textarea name="comment" id="comment" cols="30" rows="10" class="text_comment" type="text" minlength="7" maxlength="500" placeholder="Write some text here" required>Write me something...</textarea>
-                <button type="submit" class="msjButton">Get in touch</button>
-            </form>
+            <div class="coment_container">
+                <h2>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h2>
+                <form class="form_comment" action="https://formspree.io/f/xdovrwnp" method="post">
+                    <div class="names_container">
+                        <div class="names_container_input">
+                            <input type="text" id="fname" name="fname" placeholder="Full Name" minlength="4" maxlength="30" class="input_place input_comment first_name" required>
+                            <hr class="InputSeparation first_name">
+                        </div>
+                        <div class="names_container_input">
+                            <input type="text" id="fsurname" name="fsurname" placeholder="Last Name" minlength="4" maxlength="30" class="input_place input_comment last_name" required>
+                            <hr class="InputSeparation last_name">
+                        </div>
+                    </div>
+                    <br>
+                    <br>
+                    <br>
+                    <input id="faddress" name="faddress" class="input_place input_comment" placeholder="Email Address" type="email" minlength="10" maxlength="30" required>
+                    <hr class="InputSeparation">
+                    <textarea name="comment" id="comment" cols="30" rows="5" class="text_comment" type="text" minlength="7" maxlength="500" placeholder="Write some text here" required>Write me something...</textarea>
+                    <button type="submit" class="msjButton">Get in touch</button>
+                </form>
+            </div>
             <hr class="CommentSeparation">
```

Comment:
- [ ] I think it would be better if you decrease the padding of your button and adjust the font weight of the text inside. 🙂️
- [ ] Please also decrease the vertical spacing on your social icons as it is looking a bit bigger when compared to the design. 🥸️

**Your design**
![Screenshot from 2023-06-09 07-11-36](https://github.com/alexansaa/Personal_Portafolio/assets/17243325/e6b2625d-806a-4f87-8ce0-f8e2326db222)

**Figma design**
![Screenshot from 2023-06-09 07-12-01](https://github.com/alexansaa/Personal_Portafolio/assets/17243325/7035a838-f442-4003-bc6a-c4ec11effc8d)